example/tests/spec.lurk

```diff
@@ -215,7 +215,70 @@
                   (closure 1)))
 
 ;; outer-evaluate-let-no-body
-!(:assert-eq nil (let ((a 9))))
```

Comment:
@namin Do you agree that this test contradicts the spec and can be removed?

It is replaced by `!(:assert-error (let ((a 1))))`.

I think this implies that `api.lisp` is out of sync with the spec.